docs/proposals/drafts/2022023-remote-write-observatorium-api.md

```diff
@@ -0,0 +1,57 @@
+---
+toc: true
+title: 2022-02 Additional Metrics Write Endpoints
+---
+
+# Additional Metrics Write Endpoints Proposal
+
+* **Owners:**:
+  * `@marcolan018`
+
+* **Related Tickets:**
+  * `<JIRA, GH Issues>`
+
+* **Other docs:**
+  * [Exporting Metrics To Additional Consumers in RedHat Advanced Cluster Management for Kubernetes](https://docs.google.com/document/d/1t7GodF4s_V5MtFtjTeiT0UFeQuu7cuSzz9-7FEENDsQ) (internal Red Hat link)
+
+## TLDR
+
+We propose to support additional metrics write endpoints. The metrics write requests can be forwarded to additional backend endpoints, which support prometheus remote write protocol, besides thanos receivers.
+
+## Why
+Currently the observatorium API always forwards the metrics to Thanos. Given that users have multiple consumers of metric data, they have a requirement for the collected metrics to be made available to these additional consumers. Although the users can pull the metrics from Thanos side directly, but that's not a realtime solution. Also it will lead to heavy workload on Thanos if users consistently query massive metrics from it. We need a way to let observatorium API to forward the metrics to more than one targets.
```

Comment:
Perhaps capitalize Observatorium throughout?  (Except when referring the the package `observatorium`).

Reply:
fixed